packages/core/src/browser/shell/application-shell.ts

```diff
@@ -89,6 +89,9 @@ export class DockPanelRenderer implements DockLayout.IRenderer {
 
     readonly tabBarClasses: string[] = [];
 
+    private readonly _onTabBarCreated = new Emitter<TabBar<Widget>>();
+    readonly onTabBarCreated = this._onTabBarCreated.event;
```

Comment:
We also have a rule against underscores: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#no_underscore_private

Reply:
is that rule also for event Emitters? because the project is full of this patter with 
```
private readonly _onDidListener = new new Emitter<...>();
readonly onDidListener = this._onDidListener.event;
```

Reply:
"Do as I say, don't do as I do" - every parent, ever ;-)

Reply:
:D hope thats better now